scripts/build_torch_wheels.sh

```diff
@@ -63,11 +63,11 @@ function maybe_install_cuda {
       sudo sh "${CUDA_FILE}" --silent --toolkit
       rm -f "${CUDA_FILE}"
     fi
-    if [ ! -f "/usr/local/cuda/include/cudnn.h" ]; then
+    if [ ! -f "/usr/include/cudnn.h" ]; then
```

Comment:
Here we need a smarter check.
If CUDNN exists (either in `/usr/include` or `/usr/local/include`, do not install.
Undo the changes in the `install_cudnn` function. If we install, we do that in `/usr/local`.

Reply:
Done.